packages/create-nx-plugin/.eslintrc.json

```diff
@@ -3,12 +3,12 @@
   "rules": {
     "no-restricted-imports": [
       "error",
-      "@nrwl/workspace",
```

Comment:
We should probably keep the @nrwl and @nx both in eslint right?

Reply:
We can drop @nrwl when we actually remove the projects

Reply:
Autoimports may still suggest things from @nrwl packages while they are installed - that said, our repo's already on the v16 betas so they aren't installed right? This should be fine.